components/meter/meter-radial.js

```diff
@@ -44,8 +44,14 @@ class MeterRadial extends LitElement {
 		const lengthOfLine = 115; // found by approximating half the perimeter of the ellipse with radii 38 and 35
 		const progressFill = this.value / this.max * lengthOfLine;
 		const secondaryTextElement = this.text ? html`<div class="d2l-body-small d2l-meter-radial-text">${this.text}</div>` : html``;
+		let label = `${this.value} / ${this.max}, progress indicator`;
```

Comment:
This will need translation, yes?

Reply:
In speaking with @m6jones, this is a known gap. There currently any localization yet, and will need to be done. It probably makes sense to do this within a separate PR though.

Reply:
Ok sounds good.